hack/dev-push-hyperkube.sh

```diff
@@ -45,7 +45,7 @@ fi
 
 kube::build::verify_prereqs
 kube::build::build_image
-kube::build::run_build_command hack/build-go.sh cmd/hyperkube
+kube::build::run_build_command make cmd/hyperkube
```

Comment:
Shouldn't be WHAT=cmd/hyperkube ?

If not - I think I don't understand it..., can you please clarify?